ouroboros-network/src/Ouroboros/Network/PeerSelection/RootPeersDNS/DNSActions.hs

```diff
@@ -371,3 +373,13 @@ ioDNSActions =
              (Right r6, Left  e4) -> return ([e4], r6)
              (Left  e6, Right r4) -> return ([e6], r4)
              (Right r6, Right r4) -> return ([], r6 <> r4)
+
+
+--
+-- Utils
+--
+
+
+fixupTTL :: DNS.TTL -> DNS.TTL
```

Comment:
I would suggest that we map all other positive numbers below, say, 30 to 30.

There is no reason for us to have a TTL less than that - we don't want to ever churn faster than that.

Reply:
That was my original suggestion :)